### PR TITLE
Create and query multiple databases

### PR DESCRIPTION
**Pull request by SaveryIV**
In this project, I completed the day 3 activities, next are pictures of the results received in the queries:

Day3-Question1
question2
question3
question4
question5
question6
question7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+('Charmander', '2020-02-08', 0, false, -11),
+('Plantmon', '2021-11-15', 2, true, -5.7),
+('Squirtle', '1993-04-02', 3, false, -12.13),
+('Angemon', '2005-06-12', 1, true, -45),
+('Boarmon', '2005-06-07', 7, true, 20.4),
+('Blossom', '1998-10-13', 3, true, 17),
+('Ditto', '2022-05-14', 4, true, 22)
+
+/* day 3 setting foreing keys values */
+
```
Comment (V-Blaze) [inline data.sql:21]: - You have done well so far :+1: :+1: But there are a few changes and requirements you need to meet before moving to the next milestone. :+1: :waffle: 

- [x] Looking at your `data.sql` file, you do not seem to have all the data in the project requirement here. you do not have the species data here.  I'll suggest you go through the requirements one more time and fill up all the provided data here. :smile: :wave: 

- - Screenshot reference:
Screenshot from 2023-09-29 08-51-26

File: README.md [hunk 2]
```diff
@@ -61,180 +22,101 @@ After you're finished please remove all the comments and instructions!
...
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[N/A]**
+- [ ] **[N/A]**
+- [ ] **[N/A]**
```
Comment (V-Blaze) [inline README.md]: -  [x] You have done well customizing your readme file, Looking at your `future feature` section, I'll suggest you remove the `N/A` and add an appropriate feature that you are going to implement later on in this project. EG: `Optimizing query time`, `Create database diagram`. :+1: :wave:  
- [x] Also I'll suggest you remove these placeholder comments as they were added there to help guide you while you customize your readme. :airplane: :small_airplane: 

- - Screenshot reference:
Selection_011

File: README.md [hunk 3]
```diff
@@ -264,28 +143,16 @@ I would like to thank...
...
-  - [Answer_2]
+N/A
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (V-Blaze) [inline README.md:153]: - [x] Your MIT link leads to a 404 error page. I'll suggest you add a license file to your project and link it appropriately. :wave:  

- - Screenshot reference:
Selection_012

## When naming your branch or pull request title, you are supposed to give it a descriptive name that's tailored to what the branch and the requirements you are working on are all about. The bench name and pull request title `Day-3` are neither descriptive nor clear on what this branch contains. 
## I suggest you change the pull request title to something more detailed like `Create and query multiple databases` and in your subsequent project, try to give your branch a better descriptive name like `Query-multiple-tables`

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @SaveryIV 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


## Highlights
- Correct Gitflow✔
- Most queries written correctly 👋 
-  Some data inserted correctly into the database ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @SaveryIV, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- Query Multiple Tables milestone done perfectly ✔️
- Add a Screenshot of all queries to the PR ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**